pkg/cosign/tuf/client.go

```diff
@@ -212,7 +216,7 @@ func localStore(cacheRoot string) (client.LocalStore, error) {
 func embeddedLocalStore() (client.LocalStore, error) {
 	local := client.MemoryLocalStore()
 	for _, mdFilename := range []string{"root.json", "targets.json", "snapshot.json", "timestamp.json"} {
-		b, err := embeddedRootRepo.ReadFile(path.Join("repository", mdFilename))
+		b, err := embeddedRootRepo.ReadFile(filepath.Join("repository", mdFilename))
```

Comment:
maybe it's this? https://github.com/golang/go/issues/45230

Reply:
Yep, that was it! We have to use `path`.